index.bs

```diff
@@ -8215,85 +8370,102 @@ for the specific requirements that the use of
-    </dl>
+</div>
 
-If [{{Exposed}}] appears on an
-[=overloaded=] [=operation=],
+If [{{Exposed}}] appears on an [=overloaded=] [=operation=],
 then it must appear identically on all overloads.
 
-The [{{Exposed}}] extended attribute
-must not be specified on both an interface
-member and a partial interface definition the interface member is declared on.
-Similarly, the [{{Exposed}}] extended attribute
-must not be specified on both a namespace
-member and a partial namespace definition the namespace member is declared on.
-
-If [{{Exposed}}] appears an interface member, then
-the interface member's [=exposure set=]
-must be a subset of the [=exposure set=] of the interface or partial interface it's a
-member of. Similarly, if [{{Exposed}}] appears on a
-namespace member, then the namespace member's [=exposure set=] must be a
-subset of the [=exposure set=] of the
-namespace or partial namespace it's a member of.
-
-An interface's [=exposure set=]
-must be a subset of the
-[=exposure set=] of all
-of the interface's [=consequential interfaces=].
+The [{{Exposed}}] extended attribute must not be specified both on
+an [=interface member|interface=], [=mixin member|mixin=], or [=namespace member=], and on
+the [=partial interface=], [=partial mixin=], or [=partial namespace=] definition
+the [=member=] is declared on.
+
+Note: This is because [{{Exposed}}] extended attribute on partials are basicaly shorthand
+for annotating every [=member=] of the partial.
+
+If [{{Exposed}}] appears on a [=partial interface=] or [=partial namespace=],
+then the partial's [=own exposure set=] must be a subset of
+the [=exposure set=] of the partial's original [=interface=] or [=namespace=].
+
+If [{{Exposed}}] appears on an [=interface member|interface=] or [=namespace member=],
+then the [=member=]'s [=exposure set=] must be a subset
+of the [=exposure set=] of the [=interface=] or [=namespace=] it is a member of.
+
+If [{{Exposed}}] appears both on a [=partial mixin=] and its original [=mixin=],
+then the [=partial mixin=]'s [=own exposure set=]
+must be a subset of the [=mixin=]'s [=own exposure set=].
+
+If [{{Exposed}}] appears both on a [=mixin member=] and the [=mixin=] it is a member of,
+then the [=mixin members=]'s [=own exposure set=]
+must be a subset of the [=mixin=]'s [=own exposure set=].
+
+As [=mixins=] can be [=included=] by different [=interfaces=],
+the [=exposure set=] of each copy of their [=mixin members|members=] is function of
+the [=interface=] that [=includes=] them.
+If either the [=mixin member=] itself, the [=partial mixin=] it is a member of (if applicable),
+or the [=mixin=] it is a member of have an [=own exposure set=],
+then the [=mixin member=]'s [=exposure set=] is the [=set/intersection=]
+of the first of these [=own exposure set=] with the the [=host interface=]'s [=exposure set=].
+Otherwise, it is the [=host interface=]'s [=exposure set=].
```

Comment:
Some grammar issues with this paragraph, and maybe some long sentences could be cut up. Also I think it's a statement of fact about the above algorithm, and not normative; if so, then maybe making that clearer would be good.